CMakeLists.txt

```diff
@@ -2526,14 +2526,23 @@ elseif(PSP)
   endif(NOT SDL2_DISABLE_SDL2MAIN)
 
 elseif(PS2)
-  list(APPEND EXTRA_CFLAGS "-DPS2" "-D__PS2__")
+  list(APPEND EXTRA_CFLAGS "-DPS2" "-D__PS2__" "-I$ENV{PS2SDK}/ports/include")
```

Comment:
Just wondering, doesn't the ps2 cmake toolchain add `-DPS2` and `-D__PS2__` as compile flags?

Reply:
No, they have currently just `_EE` but IMO is cleaner to have `PS2` and `_PS2_`.
I will try to convince PS2DEV guys to put there these flags, and then I will remove it from here